config/config.go

```diff
@@ -188,6 +193,8 @@ type Config struct {
 	RuleFiles      []string        `yaml:"rule_files,omitempty"`
 	ScrapeConfigs  []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
 
+	RemoteWriteConfig []RemoteWriteConfig `yaml:"remote_write,omitempty"`
```

Comment:
Hm yeah, was wondering about that as well. Maybe we should have done that for other experimental parts, but now it's a bit late to introduce it. I would like that more than only depending on a "beta" marker in the docs. But ok, we can do that the same way here again.